app/components/global/layout/DisplayBasedOnScreenSize.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const HideOnMobile = styled.div`
+  @media (max-width: 480px) {
+    display: none;
+  }
+`
+
+export const ShowOnMobile = styled.div`
+  @media (min-width: 480px) {
```

Comment:
I think the way matching works, at exactly `480px` neither of these will be displayed.

Reply:
Oh! Thanks :+1: 
Will fix tomorrow

Reply:
@tamlyn What would you suggest?

I'm thinking that maybe it would be best to only use `max-width` inside a helper & then taking advantage of precedence like this:

the top submission button (desktop only) would have:
```js
${media.mobile`display: none;`}
```
but then the submission button in the footer (mobile only) would have:
```js
${media.desktop`display: none;`}
${media.desktop`display:block;`}
```

Do you guys reckon that's a good approach?

Reply:
To keep things in line with [how `grid-styled` uses breakpoints](https://github.com/rebassjs/grid#responsive-styles), I'm using `min-width` instead of `max-width`:
> Rebass Grid props accept arrays as values for mobile-first responsive styles, where the first value is for all breakpoints, then each value after is for a min-width media query from that breakpoint and up.